### ISSUE TITLE
Runtime error found, unable to use the module.

### ISSUE THREAD

**Issue by crazyyi** [drewblaisdell/monitor.io#5]
(node:13794) DeprecationWarning: Using Buffer without `new` will soon stop working. Use `new Buffer()`, or preferably `Buffer.from()`, `Buffer.allocUnsafe()` or `Buffer.alloc()` instead.
.../node_modules/buffers/index.js:185
    return this.buffers[pos.buf].get(pos.offset);
                                 ^

TypeError: this.buffers[pos.buf].get is not a function
    at Buffers.get 

I use Node.js 7.0.0 and socket.io 1.5.1

**Comment by HosseinAgha** [comment]
I don't think this project is maintained anymore for newer versions of node and socket.io

**Comment by moppymopperson** [comment]
Same issue here
